src/components/ae-slide/ae-slide.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+  <agile class="ae-slide" :class="{ fullscreen }" :options="defaults">
+    <slot />
+  </agile>
+</template>
+<script>
+import { Agile } from 'vue-agile';
+
+/**
+ * The ae-slide uses vue-agile under the hood: https://github.com/lukaszflorczak/vue-agile
+ *
+ * The option is the configuration object, where you can modify
+ * the behavior of the vue-agile.
+ *
+ * Currently this is a simple wrapper on top of the vue-agile
+ * with some stylistic changes, and acts like a interface
+ * in case in the future we will have to change / implement our
+ * own slider.
+ */
+export default {
+  name: 'ae-slide',
+  components: { Agile },
+  props: {
+    /**
+     * You can modify the behavior of the slider
+     * using the following options:
+     * @link https://github.com/lukaszflorczak/vue-agile#options
+     */
+    options: Object,
+
+    /**
+     * Creates a full screen slider, where the element tries to go
+     * 100% height and 100% width of the parent element, and places
+     * the round dots inside of the slider
+     */
+    fullscreen: Boolean,
+  },
+  computed: {
+    /**
+     * Returns the default object
+     * configuration for vue-agile
+     * @return {{}}
+     */
+    defaults() {
+      return Object.assign({
+        arrows: false,
+        infinite: false,
+      }, this.options);
+    },
+  },
+};
+</script>
+<style lang="scss" scoped>
+  @import '../../styles/globals';
+
+  .ae-slide {
+    position: relative;
+    width: 100%;
+  }
+
+  .ae-slide /deep/ .agile__slide {
+    padding: rem(20px) rem(32px);
+  }
+
+  .ae-slide /deep/ .agile__dots {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 20px;
+    margin: 0;
+    padding: 0;
+  }
+
+  .ae-slide /deep/ .agile__dots > .agile__dot {
+    @include size(10px);
+
+    list-style: none;
+    border-radius: 50%;
+    background: $color-neutral;
+    margin: 0 rem(2px);
+    transition: all $base-transition-time;
+    cursor: pointer;
+
+    > button {
+      background: none;
+      height: 100%;
+      width: 100%;
+      border: none;
+    }
+
+    &--current {
+      background: $color-primary;
+    }
+  }
+
+  .ae-slide.fullscreen {
+    height: 100%;
+  }
+
+  .ae-slide.fullscreen /deep/ .agile__list,
+  .ae-slide.fullscreen /deep/ .agile__track {
+    height: 100%;
+  }
+
+  .ae-slide.fullscreen /deep/ .agile__dots {
+    position: absolute;
+    right: 0;
+    left: 0;
+    bottom: 0;
+    height: auto;
+    padding: rem(20px) rem(32px);
+    z-index: $stack-5;
```

Comment:
why exactly `$stack-5` instead of `1` or `5`? Actually `$stack-5` is a bad name, I can't understand when I should use it.

Reply:
Any ideas for a better name for the z-index stacks?

Reply:
We should drop `stack.scss` and use just numbers if it is necessary at all.

Reply:
@davidyuk I'm okay with that, but long-term I think we need to find a standard way for z-index, as we don't want overlapping modals, popups over tooltips to be all around!

Can we merge this, and I'm going to make a pull-request to drop the `stack.scss`

Reply:
This kind of issues should be solved by order of nodes in markup instead of `z-index`, no need to write `z-index: 9999999999999999;`. For example, modals can be placed at the end of markup then they won't overlap with other elements unless they are not using z-index.
Please, check the sources of the Base app, it has minimal usage of `z-index`.